build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -1,3 +1,6 @@
+parameters:
+  extensions: []
```

Comment:
Suggest naming this something like extensionsToUnitTest to make it clear that this shouldn't include ALL extensions